rfc/rfc-000-lightweightRFCprocess.md

```diff
@@ -0,0 +1,76 @@
+# RFC-000: Lightweight RFC Process for riff
```

Comment:
rename file `rfc-0000-lightweight-rfc-process.md`

- add an extra zero to the number
- using kabob case to avoid capitalization ambiguities

Reply:
done!